### PR TITLE
Feature/#5 docker build

### PR DESCRIPTION
**Pull request by hampuslidin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
+###############################################################################
+# Backend building stage                                                      #
+###############################################################################
+FROM rust:1.66 as build-backend
+
+# Install dependencies
+RUN apt update
+RUN apt install -y gcc-arm-linux-gnueabihf
```
Comment (joellidin) [inline Dockerfile:8]: I don't know this package.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+# Create a new empty project for the backend
+RUN USER=root cargo new --bin backend
+WORKDIR /backend
+
+# Create cargo config for setting the linker command
+RUN mkdir ./.cargo
+RUN echo "[target.armv7-unknown-linux-gnueabihf]\nlinker = \"arm-linux-gnueabihf-gcc\"" > ./.cargo/config
```
Comment (joellidin) [inline Dockerfile:16]: or these configs.

**Review by joellidin — APPROVED** [review]
I guess this is meant for the raspberry pi?

File: Dockerfile (hunk 1 above)
Comment (hampuslidin) [reply Dockerfile:8]: Yes, target arch is arm64

File: Dockerfile (hunk 2 above)
Comment (hampuslidin) [reply Dockerfile:16]: This binds the previously installed package at line 8 as the linker for cargo to use.

**Pull request merged**